org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/SonarLintPostBuildListener.java

```diff
@@ -48,11 +48,11 @@
 import org.sonarlint.eclipse.ui.internal.markers.ShowIssueFlowsMarkerResolver;
 import org.sonarlint.eclipse.ui.internal.util.PlatformUtils;
 
-public class SonarLintChangeListener implements IResourceChangeListener {
+public class SonarLintPostBuildListener implements IResourceChangeListener {
 
   @Override
   public void resourceChanged(IResourceChangeEvent event) {
-    if (event.getType() == IResourceChangeEvent.POST_CHANGE) {
+    if (event.getType() == IResourceChangeEvent.POST_BUILD) {
```

Comment:
Will this work for languages that are not built, such as JavaScript?

Reply:
I did a test with JavaScript, and QA is passing. Also, documentation says the POST_BUILD event should be triggered at the time build would have been done, independently of the fact a build(er) actually run.